test/IISIntegration.FunctionalTests/Inprocess/IISTests.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
+{
+    public class IISTests : LoggedTest
+    {
+        public IISTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        [SkipIfIISExpressSchemaMissingInProcess]
+        [SkipIfNotAllowedToRunIIS]
+        public Task HelloWorld_IISExpress_Clr_X64_Standalone()
+        {
+            return HelloWorld(RuntimeFlavor.CoreClr, ApplicationType.Standalone);
+        }
+
+        [Fact]
+        [SkipIfIISExpressSchemaMissingInProcess]
+        [SkipIfNotAllowedToRunIIS]
+        public Task HelloWorld_IISExpress_Clr_X64_Portable()
+        {
+            return HelloWorld(RuntimeFlavor.CoreClr, ApplicationType.Portable);
+        }
+
+        private async Task HelloWorld(RuntimeFlavor runtimeFlavor, ApplicationType applicationType)
+        {
+            var testName = $"HelloWorld_{runtimeFlavor}";
+            using (StartLog(out var loggerFactory, testName))
```

Comment:
Please don't bring the old patterns back.

Reply:
Yeah I'll remove it; I just wanted to get it working first off. Will cleanup now.